napari/layers/image/image.py

```diff
@@ -407,7 +414,7 @@ def _data_view(self):
         """Viewable image for the current slice. (compatibility)"""
         return self._slice.image.view
 
-    def _calc_data_range(self, mode='data') -> Tuple[float, float]:
+    def _calc_data_range(self, mode='data') -> None | tuple[float, float]:
```

Comment:
`from future import __annotation__` allow to use `tuple[float, float]`?

Reply:
it seems indeed like it does yes.